src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      class="search" 
+      data-qa="big"
+    >
+      <label class="label">
+        <input 
+        class="input"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
Fix all cases
```suggestion
        <input 
          class="input"
          type="text"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
![image](https://user-images.githubusercontent.com/96409563/229997088-fb8c81ab-0cb2-4119-8944-7ce7b5fffc77.png)